packages/loadtest-bot/cloudbuild.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: gcr.io/cloud-builders/npm
+    id: "build"
+    waitFor: ["-"]
+    entrypoint: bash
+    args:
+      - "-e"
+      - "./scripts/build-function.sh"
+      - "$_DIRECTORY"
+
+  - name: gcr.io/cloud-builders/gcloud
+    id: "publish-function"
+    waitFor: ["build"]
+    entrypoint: bash
+    args:
+      - "-e"
+      - "./scripts/publish-function.sh"
+      - "$_DIRECTORY"
+      - "$PROJECT_ID"
```

Comment:
I think this is going to use our non-staging environment. Is that intended?

Reply:
`$PROJECT_ID` is determined by which project we use for Cloud Build. You can deploy to the production project if you use the production project for cloud build. You can deploy to the staging project if you use the staging project.

Reply:
Yeah, just double-checking the intent. Our CD pipeline will find this `cloudbuild.yaml`, set up a build trigger, and push this to our production project which is fine.

Reply:
Oh I didn't know it's automatically done. I don't want to deploy this to production (no harm, but it's a waist).

I think we can rename the cloudbuild.yaml to something else for opting out.